### DISCUSSION TITLE
Questions about code changes

### DISCUSSION THREAD

**Discussion by WizardKiratomato** [abizovnuralem/go2_ros2_sdk#100] [Announcements]
Dear author, thank you very much for your open source, using your new version has greatly improved compared with the one a month ago, but I can't find what you have modified by comparing the code, so I humbly ask for advice, and want to learn what can be changed to make it so changed? Looking forward to your reply! Thank you!

**Comment by abizovnuralem** [comment]
Hello! You can check git logs and see what has been changed
